axelor-human-resource/src/main/java/com/axelor/apps/hr/service/project/ProjectPlanningTimeServiceImpl.java

```diff
@@ -655,4 +655,25 @@ protected boolean isValidProjectTask(ProjectTask projectTask) {
         && projectTask.getAssignedTo() != null
         && projectTask.getAssignedTo().getEmployee() != null;
   }
+
+  @Override
+  public List<Long> getOpenProjectTaskIdList(
+      Employee employee, LocalDate fromDate, LocalDate toDate) {
+
+    return projectTaskRepo
+        .all()
+        .filter(
+            "self.status.isCompleted is false "
+                + "and self.project.manageTimeSpent is true "
+                + "and self.projectPlanningTimeList.employee = :employee "
+                + "and exists(select 1 from ProjectPlanningTime planningTime where planningTime.projectTask.id = self.id and (planningTime.startDateTime between :fromDate and :toDate or planningTime.endDateTime between :fromDate and :toDate))")
```

Comment:
Replace the last line by 
`+ "and exists(select 1 from ProjectPlanningTime planningTime where planningTime.projectTask.id = self.id and ((planningTime.startDateTime < :fromDate and planningTime.endDateTime > :toDate) or planningTime.startDateTime between :fromDate and :toDate or planningTime.endDateTime between :fromDate and :toDate))")`